www/artists/cepp/contact.html

```diff
@@ -35,7 +35,7 @@
                         <a href="contact.html">Contact</a>
                     </li>
                     <li>
-                        <a href="/">Sask Artists</a>
+                        <a href="../">Saskatchewan Artists</a>
```

Comment:
...This actually breaks the link. The original link worked fine. Check the actual website if you don't believe me.

Reply:
I will revert it to how it was before.